config.js

```diff
@@ -337,6 +339,7 @@ config.DEFAULT_S3_AUTH_METHOD = {
 //////////////////////
 
 config.LIFECYCLE_INTERVAL = 8 * 60 * 60 * 1000; // 8h
+config.LIFECYCLE_DISABLED = false;
```

Comment:
as you changed the type of this config to boolean it can be updated to ENABLED as the other BG workers, I think the DISABLED was relevant when it was a string, also it's very confusing to have negated condition